chia/util/db_wrapper.py

```diff
@@ -272,7 +272,7 @@ async def _savepoint_ctx(self) -> AsyncIterator[None]:
         await self._write_connection.execute(f"SAVEPOINT {name}")
         try:
             yield
-        except:  # noqa E722
+        except:
```

Comment:
Hmm, we've lost checking for overly broad `except`s?

Reply:
Ruff is apparently smart enough to know that re-raising the exception within the `except` block means it's fine to have the broad except.  If the `raise` wasn't in there, it would flag this.